src/imports/otp.js

```diff
@@ -84,9 +95,11 @@ export default new (class {
     }
   }
 
-  async assertInitialized() {
-    if (!this.service) {
-      throw new Error("Twilio service isn't initialized. Env vars are missing or invalid")
-    }
+  generateJWT() {
+    const token = this.jwt.sign({ foo: 'bar' }, this.jwtSecret, {
```

Comment:
juist `return this.jwt.sign(....`

Reply:
sign foo: bar??
cfverify worker is supposed to validate it

Reply:
I can't verify it on the worker since I am also dependant on what auth0 are sending on their JWT token 
the token is verified using the jsonwebtoken .verify method which is enough